test/serverspec/spec/always/paths_spec.rb

```diff
@@ -99,10 +99,3 @@
   it { should be_grouped_into 'root' }
   it { should_not be_writable.by('others') }
 end
-
-describe file('/srv/deskpro/services/messenger-api/.env') do
```

Comment:
@MattGrundy FYI - this has been broken for a while I guess

Reply:
It looks like it may be a race condition, pulling a container locally and checking, the file gets created as expected. To double check, I checked out out main, added a fairly long sleep on the earthly run between starting the container and running  and then the suite passes locally (it doesn't without the sleep). 
I'll probably need to add is-ready type checks in like we do in our product suites and get these running on our PRs but I can pick that up separately so as to not block you.